### PR TITLE
Updates test-t5x.sh to allow the specification of contrib/gpu configs which enable TE

### PR DESCRIPTION
**Pull request by terrykong**
Some code paths in the TE integration were not invoked b/c the configs used in test-t5x.sh were the upstream ones, not the GPU ones in the contrib folder. Adding this flag to switch the configs allows us to use TE layers, which we previously were not.

### CODE REVIEW

File: .github/container/test-t5x.sh [hunk 1]
```diff
@@ -14,6 +14,7 @@ usage() {
     echo "  OPTIONS                DESCRIPTION"
     echo "  -a, --additional-args  Additional gin args to pass to t5x/train.py"
     echo "  -b, --batch-size       Global batch size (REQUIRED)"
+    echo "  -c --use-gpu-configs   If provided uses contrib/gpu configs instead of top-level configs. Notably, gpu configs use adamw instead of adafactor"
```
Comment (yhtang) [inline .github/container/test-t5x.sh:17]: Perhaps call it `--use-contrib-configs` to emphasize that it is 'contribute', as GPU is always implied by our container? 😁

File: .github/container/test-t5x.sh [hunk 2]
```diff
@@ -31,6 +32,7 @@ fi
 
 ADDITIONAL_ARGS=""
 BATCH_SIZE=0
+USE_CONTRIB_CONFIGS=0
```
Comment (yhtang) [inline .github/container/test-t5x.sh:35]: Ha.. you are already using this name internally.

**Review by yhtang — CHANGES_REQUESTED** [review]

File: .github/container/test-t5x.sh (hunk 1 above)
Comment (terrykong) [reply .github/container/test-t5x.sh]: Done!

File: .github/container/test-t5x.sh (hunk 2 above)
Comment (terrykong) [reply .github/container/test-t5x.sh:35]: Yea, I think I realized that half way in and missed it in the usage. Thanks for catching

**Comment by yhtang** [comment]
Pax tests are failing for this PR. @ashors1 Could you please verify if this is expected?

**Comment by ashors1** [comment]
The Pax failure is expected. This is the same failure we have been seeing in our nightlies recently.

**Pull request merged**